desktop/cmp/form/dropdown/ComboField.js

```diff
@@ -39,11 +39,11 @@ export class ComboField extends BaseComboField {
 
     constructor(props) {
         super(props);
-        this.internalOptions = this.normalizeOptions(props.options);
+        this.setInternalOptions(this.normalizeOptions(props.options));
```

Comment:
Realize this does not relate to this change, but I am not sure why we are not just running/setting up the auto-run in the constructor.  It seems to me the current setup will cause it to be run twice and adds extra complexity.
+